buddyboss/buddyboss-platform
# Fixed Gif in Forum has play button misaligned

### All Submissions:


Fixes #1762

### How to test the changes in this Pull Request:

1. Go to Forum -> discussion
2. Make new reply with GIF and post it
3. See Play icon alignment

### Proof Screenshots or Video
https://prnt.sc/vhydwb

### Other information:
